fluentd/fluent.conf

```diff
@@ -111,9 +132,11 @@
   pos_file /var/log/node.log.pos
   tag system.var.log
   format multiline
-  format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format_firstline /^[A-Z][a-z]{2} [ |0-3][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
+  format1 /^(?<time>\S*\s+\S*\s+\S*) (?<host>\S*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
```

Comment:
This is reasonably clear if you know how the multiline format works, not so much if you don't :) Also echoing the above concerns re `\s+` as well as noting that `\S*` could theoretically match nothing (we know it will work because format_firstline matched, but still... best to be explicit). So I'd suggest another brief comment as well as minor adjustments to the regex:

```
# extract metadata from same line that matched format_firstline
format1 /^\s*(?<time>\S+\s+\S+\s+\S+)\s+(?<host>\S+)\s+(?<ident>[\w\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\:\s*(?<message>.*)$/
```